### ISSUE TITLE
Creating a Load Balancer with default backend blocks Rancher GUI

### ISSUE THREAD

**Issue by slavniyteo** [rancher/rancher#18625]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**
1. In any project or namespace create a load balancer with `Use as the default backend` selected;

**Result:** 
- Rancher GUI is unreachable;
- kube-apiserver is unreachable;
So you can not simply rollback the Ingress configuratoin.

**Why:** Since `cattle-system.rancher` balancer declares only default backend, it gets overrided by another Ingress.

To recover the cluster I had to exec into ingress-controller container at one of the nodes, rewrite nginx.conf in order to get access to Rancher GUI and then remove the Load Balancer with *default backend*. 

**Preventing:** In order to prevent losing access to Rancher GUI in the `cattle-system` namespace I created one more Load balancher with hostname set.

---

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.1.6
- Installation option (single install/HA): HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Hosted
- Machine type (cloud/VM/metal) and specifications (CPU/memory): VM
- Kubernetes version (use `kubectl version`): v1.12.4
- Docker version (use `docker version`): 18.09.2

**Comment by davidnuzik** [comment]
v2.2.0-rc12 cannot reproduce in this version.
I created an HA Rancher setup using suggestions on the Rancher docs (3 nodes, setup Amazon R53 DNS, etc).
I created a GKE cluster. On that I created an nginx workload with nodeport port 80. I created an ingress using "Use as the default backend" and successfully saved and it created. (L7 Ingress) it didn't seem to break anything.

Either I have done something wrong, or this is fixed.

Screenshot of me making the LB:
![image](https://user-images.githubusercontent.com/45179589/54942768-08997080-4eed-11e9-8691-f44f8b0aa3a0.png)
it targets my default nginx workload
